scripts/release/preparepackages.js

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env node
+
+/**
+ * @license Copyright (c) 2020-2023, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/* eslint-env node */
+
+'use strict';
+
+const { Listr } = require( 'listr2' );
+const releaseTools = require( '@ckeditor/ckeditor5-dev-release-tools' );
+const parseArguments = require( './utils/parsearguments' );
+const { PACKAGES_DIRECTORY, RELEASE_DIRECTORY } = require( './utils/constants' );
+
+const cliArguments = parseArguments( process.argv.slice( 2 ) );
+const latestVersion = releaseTools.getLastFromChangelog();
+const versionChangelog = releaseTools.getChangesForVersion( latestVersion );
+
+const tasks = new Listr( [
+	{
+		title: 'Verifying the repository.',
+		task: async () => {
+			const errors = await releaseTools.validateRepositoryToRelease( {
+				version: latestVersion,
+				changes: versionChangelog,
+				branch: 'master'
+			} );
+
+			if ( !errors.length ) {
+				return;
+			}
+
+			return Promise.reject( 'Aborted due to errors.\n' + errors.map( message => `* ${ message }` ).join( '\n' ) );
+		}
+	},
+	{
+		title: 'Updating the `#version` field.',
+		task: () => {
+			return releaseTools.updateVersions( {
+				packagesDirectory: PACKAGES_DIRECTORY,
+				version: latestVersion
+			} );
+		}
+	},
+	{
+		title: 'Copying `package-generator` packages to the release directory.',
+		task: () => {
+			return releaseTools.prepareRepository( {
+				outputDirectory: RELEASE_DIRECTORY,
+				packagesDirectory: PACKAGES_DIRECTORY,
+				packagesToCopy: cliArguments.packages
+			} );
+		}
+	},
+	{
+		title: 'Cleaning-up.',
+		task: () => {
+			return releaseTools.cleanUpPackages( {
+				packagesDirectory: RELEASE_DIRECTORY,
+				packageJsonFieldsToRemove: [
+					// TODO: Preserve `scripts.postinstall`.
+					// See: https://github.com/ckeditor/ckeditor5/issues/14318.
+					'devDependencies',
+					'depcheckIgnore'
+				]
+			} );
```

Comment:
Do we need the `postinstall` script in a package published on npm?

Reply:
Good catch with the comments.

I'm just not sure about publishing the `postinstall` hook. We are publishing it for example in react and vue integrations:
https://www.npmjs.com/package/@ckeditor/ckeditor5-react?activeTab=code
https://www.npmjs.com/package/@ckeditor/ckeditor5-vue?activeTab=code

So I'm adding it to preserve the convention.